sljit_src/sljitNativeX86_common.c

```diff
@@ -592,7 +593,11 @@ static void get_cpu_features(void)
 			feature_list |= CPU_FEATURE_LZCNT;
 	}
 
-	if ((feature_list & CPU_FEATURE_OSXSAVE) && (execute_get_xcr0_low() & 0x4) == 0)
+#if defined(SLJIT_CONFIG_X86_32) && SLJIT_CONFIG_X86_32
+	if (!(feature_list & CPU_FEATURE_SSE2))
+		feature_list &= ~(sljit_u32)CPU_FEATURE_SSE41;
```

Comment:
Who sets this? Does the compiler runs on that old hardware?

Reply:
Just like AVX2; SSE2 also needs OS support, which could be missing (ex: Debian 3).

When `gcc` targets a CPU with SSE2 it defines `__SSE2__` just like MSVC does with `_M_IX86_FP`

Reply:
The question was how is it possible to set SSE4.1 without setting SSE2. I always worry that we accidentally disables features for newer systems.